cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/RepositoryManager.java

```diff
@@ -227,8 +228,8 @@ public static void validateConfig(final SecureStore secureStore,
    * @throws SourceControlException   when failed to get the fileHash before
    *                                  push
    */
-  public String commitAndPush(final CommitMeta commitMeta,
-      final Path fileChanged)
+  public Map<Path, String> commitAndPush(final CommitMeta commitMeta,
```

Comment:
Define a class for the result instead of Map<Path, String> to better represent what the result is and easier to include more information in future.